tests/ast/data/DataFrame.unpivot.test

```diff
@@ -0,0 +1,20 @@
+## TEST CASE
+
+# TODO(SNOW-1663699): Re-enable the test.
```

Comment:
This line is no longer necessary now. Thank you!

Reply:
Removed